src/main.rs

```diff
@@ -20,6 +21,10 @@ use wrangler::settings::global_user::GlobalUser;
 use wrangler::settings::toml::TargetType;
 use wrangler::terminal::{emoji, interactive, message, styles};
 
+fn unused_code() {
+    println!("THIS IS NEVER USED EITHER");
+}
```

Comment:
just testing it out?

Reply:
yeah - want to make sure it fails in CI. it failed locally but want to double check